pkg/controllers/sidecar/config/config_test.go

```diff
@@ -186,3 +186,59 @@ func writeTempFile(t *testing.T, namePattern, content string) string {
 
 	return tmp.Name()
 }
+
+func TestScyllaArguments(t *testing.T) {
```

Comment:
Split it to multiple tests. Failure in one case shouldn't cause failure in cases where regexp works fine.
You can use table tests: https://github.com/golang/go/wiki/TableDrivenTests

Reply:
done